ambient.d.ts

```diff
@@ -319,6 +344,56 @@ declare module '@mojaloop/central-services-error-handling' {
   export function ReformatFSPIOPError(error: any, apiErrorCode?: any, replyTo?: any, extensions?: any): FSPIOPError
   export function CreateFSPIOPError(apiErrorCode?: any, message?: any, cause?: any, replyTo?: any, extensions?: any, useDescriptionAsMessage?: boolean): FSPIOPError
 }
+declare module '@mojaloop/central-services-stream' {
+  import { EventEmitter } from 'events';
+  import { EventActionEnum, EventTypeEnum } from '@mojaloop/central-services-shared';
+  export interface KafkaConsumerConfig {
+    eventType: EventTypeEnum,
+    eventAction: EventActionEnum,
+    options: {
+      mode: number,
+      batchSize: number,
+      pollFrequency: number,
+      recursiveTimeout: number,
+      messageCharset: string,
+      messageAsJSON: boolean,
+      sync: boolean
+      consumeTimeout: number
+    },
+    rdkafkaConf: {
+      'client.id': string,
+      'group.id': string
+      'metadata.broker.list': string,
+      'socket.keepalive.enable': boolean
+    },
+    topicConf: {
+      'auto.offset.reset': string
+    }
+  }
+
+  // TODO: figure out a better example of a message - we may need to just do some stringifying
+  type Message = any;
+  interface GetMetadataResult {
+    topics: Array<{
+      name: string
+    }>
+  }
+  class Consumer extends EventEmitter {
+    constructor(topics: Array<any>, config: KafkaConsumerConfig)
+    connect(): Promise<boolean>;
+    consume(workDoneCb: (error: Error, payload: Message | Array<Message>) => Promise<void>): void
+    disconnect(cb: () => any): void;
+    getMetadata(options: any, cb: (err: any, result: GetMetadataResult) => any): void;
```

Comment:
let not use any - unknown if needed, we can reverse-engineer the signature of the callback function from here: https://github.com/mojaloop/central-services-stream/blob/65913b6cbbcdcf1fe60df885f007fe5e0000aab3/test/unit/kafka/KafkaStub.js#L112